benchmark/cases/react_speed_in_loop.py

```diff
@@ -35,8 +35,12 @@ def react_speed_in_loop(benchmark: BenchmarkControl, inbox_amount=1000) -> None:
 
     :return: None
     """
-    skill_definition = {"handlers": {"dummy_handler": DummyHandler}}
-    aea_test_wrapper = AEATestWrapper(name="dummy agent", skills=[skill_definition],)
+    aea_test_wrapper = AEATestWrapper(
+        name="dummy agent",
+        components=[
+            AEATestWrapper.make_skill(handlers={"dummy_handler": DummyHandler})
```

Comment:
I think the key needs to be the full public id in str form...but worth checking

Reply:
no, we use just plain names for skill components.

Reply:
Yes correct.